.travis.yml

```diff
@@ -18,6 +18,7 @@ jobs:
         - KERAS=2.2.4
         - PYTORCH=1.2.0
         - TORCHVISION=0.4.0
+        - PYTORCH_LIGHTNING=0.7.5
```

Comment:
Can you move `PYTORCH_LIGHTNING' one line up, so it aligns with `PYTORCH`?

Reply:
sure